### PR TITLE
Clarified instructions on how/when to run the scripts

### PR DESCRIPTION
**Pull request by xenoworf**
### Summary
I was very confused by the script documentation, so after talking with @jeremyfuksa I've moved the doc into the terra-icon package readme and renamed a couple npm commands to make more sense. This will make it easier for the next dev.

### Deployment Link
TBD

### Testing
I ran `migrate-ocs-icons` and `generate-icons` manually to make sure they are not broken.

### CODE REVIEW

**Review by jeremyfuksa — APPROVED** [review]

File: packages/terra-icon/README.md [hunk 1]
```diff
@@ -10,10 +10,22 @@ The terra-icon component is used to visually represent a literal or symbolic obj
...
 - [LICENSE](#license)
 
-## Getting Started
+## Using Terra Icon
 
 - Install from [npmjs](https://www.npmjs.com): `npm install terra-icon`
 
+## Adding new icons from the One Cerner Style Icons repo
+
+    npm run migrate-ocs-icons
```
Comment (sdadn) [inline packages/terra-icon/README.md:19]: nit: should this be in a code block (```)

File: packages/terra-icon/scripts/src/README.md [hunk 2]
```diff
@@ -1,10 +1 @@
...
-
-```
-npm run migrate-cerner-one-icons
-```
-* npm run compilescripts
-* npm run migrate-csv
-* npm run migrate-svg
-* npm run generate-icon
-* npm run generate-example
+Each script has its own readme. Instructions on when and how to use these scripts are in the terra-icons readme.
```
Comment (sdadn) [inline packages/terra-icon/scripts/src/README.md:1]:
```suggestion
Each script has its own readme. Instructions on when and how to use these scripts are in the [terra-icons readme](insert link to readme).
```

File: packages/terra-icon/README.md (hunk 1 above)
Comment (xenoworf) [reply packages/terra-icon/README.md:19]: Nope, indenting by one tab is the same effect. [Check it out in action](https://github.com/cerner/terra-core/blob/update-script-instructions/packages/terra-icon/README.md).

File: packages/terra-icon/scripts/src/README.md (hunk 2 above)
Comment (xenoworf) [reply packages/terra-icon/scripts/src/README.md:1]: Fixed in 9115d408060a5416be60684ef6246049fedc0023

**Review by sdadn — APPROVED** [review]

**Review by supreethmr — APPROVED** [review]

**Review by benbcai — APPROVED** [review]

**Pull request merged**